src/components/ProfileInfo/ProfileInfo.tsx

```diff
@@ -0,0 +1,78 @@
+import Button from 'components/core/Button/Button';
+import ProfileInfoDialog from 'components/ProfileInfoDialog/ProfileInfoDialog';
+import UserModel from 'models/UserModel';
+import React, { useEffect, useState } from 'react';
+import { useLocation, useNavigate } from 'react-router-dom';
+import tokenService from 'services/tokenService';
+import userService from 'services/userService';
+import classes from './ProfileInfo.module.scss';
+
+const ProfileInfo = () => {
+  const [user, setUser] = useState<UserModel>({} as UserModel);
+  const location = useLocation();
+  const id: number = parseInt(location.pathname.split('/')[2]);
+  const [modify, setModify] = useState<Boolean>(false);
+  const { isRoleAdmin } = tokenService;
+  const isAdmin = isRoleAdmin();
+  const { getUserById } = userService;
+  const { deleteUser } = userService;
+  const navigate = useNavigate();
```

Comment:
Also one suggestion is to try to give some structure when assigning variables. For example first do all useState, then other hooks, then service...